bundlegen/core/library_matching.py

```diff
@@ -51,7 +52,16 @@ def _remove_from_rootfs(self, rootfs_filepath):
         os.remove(rootfs_filepath)
         if (link != rootfs_filepath):
             os.remove(link)
-
+    # ==========================================================================
+    def _create_mount_point(self, fullPath):
+        """Create mount point (empty file) in provided path.
+        """
+        # Create the directory if doesn't exist
+        directory = os.path.dirname(fullPath)
+        if not os.path.exists(directory):
+            os.makedirs(directory, 0o755)
+        open(fullPath, mode = 'x')
+        os.chmod(fullPath, 0o777)
```

Comment:
777 seems a bit extreme - can this not be set to something tighter?

Reply:
done